svc/auth/transport.go

```diff
@@ -378,6 +378,15 @@ func codeAndMessageFrom(err error) (int, interface{}) {
 		return http.StatusBadRequest, err.Error()
 	}
 
+	if errors.Is(err, ErrKYCInProgress) ||
+		errors.Is(err, ErrUserIsDisabled) {
+		return http.StatusForbidden, err.Error()
+	}
+
+	if errors.Is(err, ErrKYCNeeded) {
+		return http.StatusProxyAuthRequired, err.Error()
+	}
+
```

Comment:
Move the handling of KYC errors to the function `httpencoder.CodeAndMessageFrom`.
In auth service, they will never happen.